examples/developer_guide/3_simple_cpp_stage/.gitattributes

```diff
@@ -0,0 +1 @@
+src/simple_cpp_stage/_version.py export-subst
```

Comment:
Just to be safe, remove this change, and then run `versioneer install --vendor` and see if it re-adds this.

Reply:
Confirmed it re-adds it. 
Also the mystery of the `versioneer.py` copy in the root of the repo is solved. 

When you run `versioneer install --vendor` it makes a copy of `versioneer.py` in the repo, allowing the package to not be a runtime or build dependency.

Running `versioneer install --no-vendor` doesn't create the copy.